repo/packages/M/minio/11/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "packagingVersion": "3.0",
+  "name": "minio",
+  "version": "0.0.12-RELEASE.2018-04-04T05-20-54Z",
+  "minDcosReleaseVersion": "1.8",
+  "scm": "https://github.com/minio/minio.git",
+  "maintainer": "dev@minio.io",
+  "website": "https://www.minio.io",
+  "framework": false,
+  "description": "Minio is an object storage server for cloud native applications. To deploy at scale, use DC/OS to orchestrate Minio containers one per tenant.",
+  "tags": [
+    "data",
+    "storage",
+    "filesystem"
+  ],
+  "preInstallNotes": "This is a community package. Community packages are unverified and unreviewed content from the community. Launch as many Minio instances as required referring to the documentation here https://github.com/dcos/examples/blob/master/minio/1.9/README.md",
```

Comment:
You may remove this line: `This is a community package. Community packages are unverified and unreviewed content from the community.` The section `Community Packages` in Universe says that now

Reply:
https://github.com/dcos/examples/blob/master/minio/1.9/README.md

Add an updated 1.11 example